### ISSUE TITLE
Parallel 'flows'

### ISSUE THREAD

**Issue by danjamesmay** [artilleryio/artillery#929]
Hi,

I noticed you support parallel 'flows' in #376, but this is [undocumented](https://artillery.io/docs/guides/guides/http-reference.html).

Is this still supported in the [latest published npm version of Artillery](https://www.npmjs.com/package/artillery) (1.6.1)?

Thanks

**Comment by hassy** [comment]
yes it is! and it's probably time to move the feature from "experimental" to "fully documented" :)

**Comment by brogerson-rbcv** [comment]
Is the docs site open source? I'm about to use this feature and I don't mind contributing a section to the docs (once I understand how it works 😅).

**Comment by jakebiesinger** [comment]
Adding my +1 here... seems like this is a really common use case (simulating browser behavior) and I'd written hundreds of lines of scenarios before I happened upon this issue and found this functionality.

Also happy to contribute to the docs... Also having written several processors and plugins recently, I find many of the docs to be a bit superficial and would be happy to improve things if there's a way to do that.
